_docs_integrate/establish-relationships.md

```diff
@@ -112,6 +112,13 @@ If an App user wants to initiate a Relationship with the templator, the App user
 
 After the templator has created a RelationshipTemplate and made it available to another Identity, this Identity can use it to initiate a Relationship with the templator. For this reason, this other Identity is also referred to below as the initiator.
 
+### Check the Initiability of a Relationship
```

Comment:
wtb "Check if the Relationship can be established"?

Reply:
What's wrong with the current wording? ;) I like it better as it is now because it is shorter and contains the word "initiate" (which I feel is more appropriate wording in this "Initiate a Relationship" section of the guide).

Reply:
I don't like the word `Initiability` that is never used again..

Reply:
> is more appropriate wording in this "Initiate a Relationship"

then rename this to establish

Reply:
I have never heard "initiability" before. Linguee doesn't know it either and it is also underlined in red here.

Reply:
> I have never heard "initiability" before. Linguee doesn't know it either and it is also underlined in red here.

yeah, and this :D

Reply:
Good bye "Initiability", we (or I 🤣) will miss you. 💔